lib/senders/sms.js

```diff
@@ -32,60 +26,49 @@ module.exports = function (log, translator, templates, smsConfig) {
         acceptLanguage: acceptLanguage
       })
 
-      return P.resolve()
-        .then(function () {
-          var message = getMessage(messageId, acceptLanguage)
-
-          return sendSms(senderId, phoneNumber, message.trim())
+      var message = getMessage(messageId, acceptLanguage)
```

Comment:
Fwiw, `getMessage` throws so we want it to stay inside the promise chain to convert that to a rejection if it happens.